ui/desktop/app/styles/app.scss

```diff
@@ -541,6 +541,25 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   }
 }
 
+// Proxy URL
+.proxy-url-container {
+  display: flex;
+  gap: 28px;
+  justify-content: flex-start;
+
+  .copy-address {
+    max-width: 65%;
+  }
+
+  .copy-port {
+    width: 120px;
+  }
+
+  .copy-ssh-command {
+    max-width: 75%;
+  }
+}
+
```

Comment:
I think we can get the inline style we were talking about if we just use
```
.proxy-url-container {
   ...container styles

  .hds-copy-snippet {
    width: auto;
  }
}
```

Nice and simple at least 😄

Reply:
Sorry, I forgot that doesn't get us the shrinking truncation on the address. Maybe it's more flex-box then.